modules/apps/jaggery/emm/src/emm/modules/sso.js

```diff
@@ -87,11 +87,11 @@ var log = new Log();
             sso.log.debug("Logout request recieved from session id ::: " + sso.sessionId);
```

Comment:
there are two separate log options one is for sso base logs and the other is to non sso related loggings